core/assets/bundles/bundle_zh_CN.properties

```
@@ -1214,7 +1214,7 @@ block.shock-mine.name = 脉冲地雷
 block.overdrive-projector.name = 超速投影器
 block.force-projector.name = 力墙投影器
 block.arc.name = 电弧
-block.rtg-generator.name = RTG 发电机
+block.rtg-generator.name = 放射性同位素热电发电机
```

```suggestion
block.rtg-generator.name = 放射性同位素热电机
```

”放射性同位素热电机“ is a shorter but accurate translation, since the word "Radioisotope" only translates to "放射性同位素", I guess we don't have much of a choice

---

同位素发电机能映射到放射性同位素热电机

---

但是热电机又可能不能明白是发电机

---

I guess to use “发电机” or “热电机” isn't a big issue, either is fine, it's just that “热电机” is a scientifically used term